docs/contracts/overview/NFT/customise-transfer-behaviour.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Customize transfer behaviour
+sidebar_position: 2
+---
+
+# Customize transfer behaviour
+
+The `LSP8IdenfitiableDigitalAsset` contract implementation includes the `_beforeTokenTransfer` and `_afterTokenTransfer` functions that offer the ability to specify custom logic that can run before or after the token transfer has happen (= before or after the balances in the contract state have been updated).
```

Comment:
Should we give code examples for those functions?

Reply:
I will but haven't found a good example so far to showcase like in LSP7. I will do in a separate PR